crates/deltalake-core/tests/command_restore.rs

```diff
@@ -128,7 +131,20 @@ async fn test_restore_by_datetime() -> Result<(), Box<dyn Error>> {
         .await?;
     let timestamp = meta.last_modified.timestamp_millis();
     let naive = NaiveDateTime::from_timestamp_millis(timestamp).unwrap();
-    let datetime: DateTime<Utc> = Utc.from_utc_datetime(&naive);
+    let datetime: DateTime<Utc> = Utc.from_utc_datetime(&naive); // - Duration::hours(1);
```

Comment:
Is this still relevant after your fix of the test?

Reply:
Oh yeah, missed that, will delete